crates/core/src/config/wow.rs

```diff
@@ -49,6 +49,14 @@ impl Flavor {
         }
     }
 
+    /// Returns flavor `String` in Tukui format
+    pub(crate) fn tukui_format(self) -> String {
```

Comment:
Are we using this anywhere other than building the API URL? If not, we might want to move this to a helper in that tukui backend module and change `classic` to `classic-addon` and reference this for the classic match as well

Reply:
Yeah good point, also changed for classic addons.